content/agent/prometheus.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Prometheus Check
+kind: documentation
+---
+
+## Overview
+
+This page details how to collect metrics and events from a Prometheus data source by writing an Prometheus Check, a Python plug-in to the Datadog Agent. This page looks first at the `PrometheusCheck` interface, and then propose a simple Prometheus Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/check.py).
```

Comment:
nit s/an Prometheus Check/a Prometheus Check/
I'd remove "a Python plug-in to the Datadog Agent", as it results in an awkward sentence. All custom check are a Python plug-in to the Datadog Agent.

Reply:
👍